### DISCUSSION TITLE
How to import existing table in directus 9

### DISCUSSION THREAD

**Discussion by devistar** [directus/directus#8798] [Archive]
How to import existing table in directus 9 ?

**Comment by joselcvarela** [comment]
Hello @devistar ,
Do you have already Directus installed and configured with your database?

**Reply by devistar** [reply]
Yes but I can't see a way to import existing table into collections

**Reply by joselcvarela** [reply]
Make a backup of your database and try the following command:
`npx directus bootstrap`

**Reply by devistar** [reply]
It doesn't work. By existing table, I mean a table created manually.

**Reply by devistar** [reply]
I found the reason. My table didn't have a primary key. With a primary key, table is recongnized.

**Comment by johannesschobel** [comment]
Dear all,

i have a similar issue. I have a fully configured "base image" of directus (i.e., a set of pre-defined collections) that i want to host multiple times. Therefore, i would like to "copy / paste" this image to new instances.

Can i achieve this via the admin panel? Because actually i don't have access to the console (i.e., run `npm / npx` commands) or underlying database tables.

All the best

**Reply by azrikahar** [reply]
Hi, can you share more detail on how the "a set of pre-defined collections" works at the moment?

> Can i achieve this via the admin panel?

Currently there is an actual way to do this via admin panel, but [Schema Snapshot](https://docs.directus.io/reference/command-line-interface/#migrate-schema-to-a-different-environment) (just in case you're not aware of it) seems like what you need. However yea there's the issue of you don't have access to the console nor the database tables 🤔
